strings_lab.js

```diff
@@ -0,0 +1,51 @@
+//Question 1
```

Comment:
I can see that you understand properties and methods for string data type but I also saw instance of hard coding. Try to avoid that in future assignments and also try to do bonus questions next time. Bonus questions are good practice to improve technical mastery.